src/cljs/chronograph_web/pages/organization/events.cljs

```diff
@@ -0,0 +1,30 @@
+(ns chronograph-web.pages.organization.events
+  (:require [re-frame.core :as rf]
+            [chronograph-web.http :as http]))
+
+
+(def ^:private get-organization-uri
+  "/api/organizations")
```

Comment:
Missing trailing `/` or, you should add it to the `str` below.

Reply:
Ick! Fixed!